extensions/amp-carousel/validator-amp-carousel.protoascii

```diff
@@ -43,6 +43,7 @@ tags: {  # <amp-carousel>
   attrs: {
     name: "autoplay"
     value: ""
+    value_regex: "[0-9]+"
```

Comment:
For `attrs` it's considered a [oneof](https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L94) of different types of values. Some can be repeated like `value`. In this case I'd recommend `value_regex: "(|[0-9]+)"` since it should allow both empty and some numeric.

For limited sets repeating value is sometimes easier to read than regex, e.g.:
```
value: "_top"
value: "_blank"
```
instead of
`value_regex: "(_top|_blank)"`

Reply:
Made the change @honeybadgerdontcare, also added a validator test for this.